Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -1363,7 +1363,7 @@ internal static void NetworkLateUpdate()
             if (active)
             {
                 // go through all connections
-                foreach (NetworkConnectionToClient connection in connections.Values)
+                foreach (NetworkConnectionToClient connection in connections.Values.ToList())
```

Comment:
This is a GC allocation per frame now. This does not seem ideal.
Wouldn't it be better to work with a cached list and use `Clear()` and `AddRange()` to achieve similar results?